addons/project/static/src/views/project_calendar/project_calendar_model.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { CalendarModel } from '@web/views/calendar/calendar_model';
+
+export class ProjectCalendarModel extends CalendarModel {
+    /**
+     * @override
+     * @returns {String}
+     */
+    defaultFilterLabel() {
+        return this.env._t("Private");
```

Comment:
But wait, this doesn't take the field into account. What if `makeFilterDynamic` is called on anything else than `project_id`?